utils.lua

```diff
@@ -6,3 +6,12 @@ function manhattan(p, q)
    end
    return s
 end
+
+function clamp(character)
+    character.position.x = math.Clamp(character.position.x, character.box_width / 2, love.graphics.getWidth() - character.box_width / 2)
```

Comment:
```suggestion
    character.position.x = math.Clamp(character.position.x, character.box_width / 2, CONF_SCREEN_WIDTH - character.box_width / 2)
```

essa nova constante global deixa tudo na base do 1080p, e a gente escala automaticamente na hora do rendering